### PR TITLE
update to crate 0.52.4 stable

### PR DESCRIPTION
**Pull request by mikethebeer**
* adapt discussed improvements related to https://github.com/docker-library/official-images/pull/1230 on dockerfile

### CODE REVIEW

File: library/crate [hunk 1]
```diff
@@ -10,5 +10,5 @@
 latest: git://github.com/crate/docker-crate@d8ef1163d5937083a6cb8831732822d0f5d3cb84
```
Comment (yosifkit) [inline library/crate]: Looks like `latest` is still the old commit :wink:.

Comment (mikethebeer) [inline library/crate]: @yosifkit  thanks :santa:

**Comment by tianon** [comment]
LGTM (https://github.com/crate/docker-crate/compare/d8ef1163d5937083a6cb8831732822d0f5d3cb84...cce8f796ba8936250eb380235cde47be494d1e95)

**Comment by yosifkit** [comment]
LGTM, Build test of #1246; 0b29933 (`crate`):

```console
$ bashbrew build "crate"
Cloning crate (git://github.com/crate/docker-crate) ...
Processing crate:latest ...
Processing crate:0.51 ...
Processing crate:0.51.7 ...
Processing crate:0.52 ...
Processing crate:0.52.4 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing crate:latest
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing crate:0.51
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
```

**Pull request merged**